doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -279,6 +279,56 @@ Matter
 
 The issues in this section are related to the :ref:`ug_matter` protocol.
 
+.. rst-class:: v2.4.1 v2-4-0 v2-3-0 v2-2-0
+
+KRKNWK-17535: The application core can crash on nRF5340 after the OTA firmware update finishes if the factory data module is enabled.
+  In the initialization method of the factory data module, the factory data partition and a part of the application image is restricted by Fprotect, which makes it impossible to confirm the new image in the Matter thread.
+  Instead, the confirmation must be performed before the factory data module is initialized.
+
+  **Affected platforms:** nRF5340
+
+  **Workaround:** Complete the following steps:
+  
+  1. Manually cherry-pick and apply the commit with the fix to ``sdk-connectedhomeip`` (commit hash: ``TODO``).
+  2. Add the following lines to the :file:`samples/matter/common/src/ota_util.cpp`:
+
+  .. code-block::
+
+    #include <platform/CHIPDeviceLayer.h>
+    #include <zephyr/dfu/mcuboot.h>
+
+    CHIP_ERROR OtaConfirmNewImage()
+    {
+      CHIP_ERROR err = CHIP_NO_ERROR;
+      OTAImageProcessorImpl &imageProcessor = GetOTAImageProcessor();
+      if (imageProcessor.IsFirstImageRun()) {
+        CHIP_ERROR err = System::MapErrorZephyr(boot_write_img_confirmed());
+        if (CHIP_NO_ERROR == err) {
+          imageProcessor.SetImageConfirmed();
+        }
+      }
+      ChipLogError(SoftwareUpdate, "Failed to confirm firmware image, it will be reverted on the next boot");
+      return err;
+    }
+
+  3. Add the following line to the :file:`samples/matter/common/src/ota_util.h`:
+
+  .. code-block::
+
+    CHIP_ERROR OtaConfirmNewImage();
+
+  4. Add the following lines to the ``AppTask::Init()`` method in the :file:`app_task.cpp` file located in a sample directory before initialization of the factory data module (``mFactoryDataProvider.Init()``):
+
+  .. code-block::
+
+    #ifdef CONFIG_CHIP_OTA_REQUESTOR
+      /* OTA image confirmation must be done before the factory data init. */
+      err = OtaConfirmNewImage();
+      if (err != CHIP_NO_ERROR) {
+        return err;
+      }
+    #endif
```

Comment:
```suggestion
  1. Manually cherry-pick and apply the commit with the fix to ``sdk-connectedhomeip`` (commit hash: ``TODO``).
  #. Add the following lines to the :file:`samples/matter/common/src/ota_util.cpp`:

     .. code-block::

        #include <platform/CHIPDeviceLayer.h>
        #include <zephyr/dfu/mcuboot.h>

        CHIP_ERROR OtaConfirmNewImage()
        {
          CHIP_ERROR err = CHIP_NO_ERROR;
          OTAImageProcessorImpl &imageProcessor = GetOTAImageProcessor();
          if (imageProcessor.IsFirstImageRun()) {
            CHIP_ERROR err = System::MapErrorZephyr(boot_write_img_confirmed());
            if (CHIP_NO_ERROR == err) {
              imageProcessor.SetImageConfirmed();
            }
          }
          ChipLogError(SoftwareUpdate, "Failed to confirm firmware image, it will be reverted on the next boot");
          return err;
        }

  #. Add the following line to the :file:`samples/matter/common/src/ota_util.h`:

     .. code-block::

        CHIP_ERROR OtaConfirmNewImage();

  #. Add the following lines to the ``AppTask::Init()`` method in the :file:`app_task.cpp` file located in a sample directory before initialization of the factory data module (``mFactoryDataProvider.Init()``):

     .. code-block::

        #ifdef CONFIG_CHIP_OTA_REQUESTOR
          /* OTA image confirmation must be done before the factory data init. */
          err = OtaConfirmNewImage();
          if (err != CHIP_NO_ERROR) {
            return err;
          }
        #endif
```
Content in following lines needs to be aligned to content on the first line, so the `..` for the code block needs to start with the same indent as the text for its entry (basically, three spaces for numbered entries, two spaces for bullets). The same applies to code blocks, so the contents of a code block has to have the same indent as the `code-block` text (so three spaces).